src/flamenco/runtime/program/fd_bpf_loader_program.c

```diff
@@ -37,6 +37,72 @@ static void __attribute__((destructor)) free_buf(void) {
 #define PROGRAMDATA_METADATA_SIZE        (45UL  ) /* UpgradeableLoaderState::size_of_programdata_metadata() */
 #define SIZE_OF_UNINITIALIZED            (4UL   ) /* UpgradeableLoaderState::size_of_uninitialized() */
 
+static inline int
+program_error_to_instr_error( ulong  err, 
```

Comment:
Could we add a permalink for this mapping?  🙏 

https://github.com/anza-xyz/agave/blob/ced98f1ebe73f7e9691308afa757323003ff744f/sdk/program/src/program_error.rs#L167